### PR TITLE
Intergrate price deviation module into monitor

### PR DESCRIPTION
**Pull request by chrismaree**
This PR integrates the price deviation monitor module into the monitor bot.

### CODE REVIEW

File: monitors/index.js [hunk 1]
```diff
@@ -30,13 +31,23 @@ const ExpandedERC20 = artifacts.require("ExpandedERC20");
...
-async function run(address, shouldPoll, botMonitorObject, walletMonitorObject, pollingDelay, priceFeedConfig) {
+async function run(
+  address,
+  shouldPoll,
+  botMonitorObject,
+  walletMonitorObject,
+  syntheticPegMonitorObject,
+  pollingDelay,
+  uniswapPriceFeedConfig,
+  medianizerPriceFeedConfig
```
Comment (mrice32) [inline monitors/index.js:42]: nit: we might want to name these a little more generically -- like primary and secondary price feed? Or primary and reference? Not a strong preference, though -- up to you.

File: monitors/index.js [hunk 2]
```diff
@@ -119,31 +153,46 @@ async function run(address, shouldPoll, botMonitorObject, walletMonitorObject, p
...
+      throw "Bad environment variables! Specify an `EMP_ADDRESS` for the location of the expiring Multi Party.";
     }
 
     if (!process.env.BOT_MONITOR_OBJECT || !process.env.WALLET_MONITOR_OBJECT) {
-      throw new Error("Bad input arg! Specify a `BOT_MONITOR_OBJECT` & `WALLET_MONITOR_OBJECT` to track.");
+      throw "Bad input arg! Specify a `BOT_MONITOR_OBJECT` & `WALLET_MONITOR_OBJECT` to track.";
     }
 
-    const pollingDelay = process.env.POLLING_DELAY ? process.env.POLLING_DELAY : 10_000;
+    const pollingDelay = process.env.POLLING_DELAY ? process.env.POLLING_DELAY : 10000;
```
Comment (mrice32) [inline monitors/index.js:163]: Why remove the `10_000` syntax? Just curious

File: monitors/index.js [hunk 3]
```diff
@@ -119,31 +153,46 @@ async function run(address, shouldPoll, botMonitorObject, walletMonitorObject, p
...
     // Bot objects to monitor. For each bot specify a name, address and the thresholds to monitor.
     const botMonitorObject = JSON.parse(process.env.BOT_MONITOR_OBJECT);
 
     // Wallet objects to monitor.
     const walletMonitorObject = JSON.parse(process.env.WALLET_MONITOR_OBJECT);
 
-    if (!process.env.PRICE_FEED_CONFIG) {
-      throw new Error("Bad input arg! Specify `PRICE_FEED_CONFIG` to define the price feed settings");
+    if (!process.env.UNISWAP_PRICE_FEED_CONFIG || !process.env.MEDIANIZER_PRICE_FEED_CONFIG) {
+      throw "Bad input arg! Specify `PRICE_FEED_CONFIG` and `MEDIANIZER_PRICE_FEED_CONFIG` to define the price feed settings.";
```
Comment (mrice32) [inline monitors/index.js:172]: Do we have a preference between throwing a string vs an `Error` object? Are there reasons to prefer one over the other?

cc @nicholaspai

Comment (nicholaspai) [reply monitors/index.js:172]: My default has been to throw explicitly an `Error` object as its more explicit FYI

File: monitors/index.js (hunk 1 above)
Comment (chrismaree) [reply monitors/index.js:42]: We _could_ do this but I think that for now, it's probably easier to the name of the reference as specific as possible so there is no opportunity for confusion. Prehaps a good name would be `synethicMarketPriceFeed` and `identifierMarketPriceFeed` but for now, I'm favoring explicit naming.

File: monitors/index.js (hunk 2 above)
Comment (chrismaree) [reply monitors/index.js:163]: It felt cleaner. Also the underscore does not work with GCP. When I set the delay to `30_000` in GCP it was fed in as `30` and lost the remaining part of the number and so the bots were polling every 30ms! Removing it here makes it consistent with the GCP env config.

File: monitors/index.js (hunk 3 above)
Comment (chrismaree) [reply monitors/index.js:172]: In this [PR](https://github.com/UMAprotocol/protocol/pull/1450) I refactored the bots to how a string, not an `Error` object. We should have a convention for this.

**Pull request merged**